packages/experimental/surface/src/plugins/ClientPlugin.tsx

```diff
@@ -0,0 +1,48 @@
+//
+// Copyright 2023 DXOS.org
+//
+
+import React from 'react';
+
+import { MulticastObservable } from '@dxos/async';
+import { Client, ClientProvider } from '@dxos/react-client';
+
+import { definePlugin } from '../framework';
+import { GraphPluginProvides } from './ListViewPlugin';
+
+export type ClientPluginProvides = GraphPluginProvides & {
+  client: Client;
+};
+
+export const ClientPlugin = definePlugin<{}, ClientPluginProvides>({
+  meta: {
+    id: 'dxos:ClientPlugin'
+  },
+  init: async () => {
+    const client = new Client();
+    await client.initialize();
+
+    return {
+      client,
+      context: ({ children }) => <ClientProvider client={client}>{children}</ClientProvider>,
+      graph: {
```

Comment:
this feels like a cyclic dep. client is really far down our stack, but this is "providing" very high level (opinionated) graph plugin semantics?